pkg/manager/manager.go

```diff
@@ -112,12 +114,17 @@ type Options struct {
 	// For namespaced resources the cache will only hold objects from the desired namespace.
 	Namespace string
 
+	// MetricsBindAddress is the TCP address that the controller should bind to
+	// for serving prometheus metrics
```

Comment:
We should probably mention the default, that metrics will be served on ":8080". 
And that to stop serving metrics this should be set to "0".

I know it's mentioned in `pkg/metrics/listener.go` but this is the main knob that users will tweak, so we should probably clarify that behavior here.